source/geod24/LocalRest.d

```diff
@@ -282,6 +282,23 @@ public void sleep (Duration timeout)
     cond.wait(timeout);
 }
 
+/// Ditto
+public void setTimer (Duration timeout, void delegate() dg,
```

Comment:
This function should be documented somewhere. For example, in the module documentation.